packages/e2e-tests/test/e2e.spec.ts

```diff
@@ -700,7 +700,7 @@ describe('e2e', function () {
             validator: {
               $and: [
                 { phone: { $type: "string" } },
-                { email: { $regex: /@mongodb\.com$/ } },
+                { email: { $regex: /@mongodb.com$/ } },
```

Comment:
Hmmm, this shouldn't be an unnecessary escape I think, you need to escape the dot if you want to match it exactly and not "everything" 🤔

Reply:
yeah I'll revert that one and just ignore the line.